_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -24,21 +25,26 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names][accessible name] that do not only consist of [whitespace](#whitespace).
+This rule applies to any set of any two or more HTML or SVG elements which
 
-**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name][].
+- are inside the [flat trees][flat tree] (work in progress) of one or more [documents][document] of the same [set of browsing context related documents][]; and
```

Comment:
What's this "work in progress" thing doing here? If this PR is WIP, please put that in the name, or make it a draft PR.

Reply:
The CSS Scoping Module Level 1 (as well as a few other specs) are only WiP and require user to "Don’t cite this document other than as work in progress."

See #700 and #716 for more on that, this has also been discussed on an ACT-R call in July (the 25th, I guess).